src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] I suggest you make your functional App.js component `class-based`  because it is one of the learning purposes of this milestone.